modules/siddhi-core/src/test/java/org/wso2/siddhi/core/store/StoreQueryTableTestCase.java

```diff
@@ -446,4 +448,76 @@ public void test11() throws InterruptedException {
         AssertJUnit.assertEquals(100L, events[0].getData()[2]);
         AssertJUnit.assertEquals(100L, events[1].getData()[2]);
     }
+
+    @Test
+    public void test12() throws InterruptedException {
+        log.info("Test12 - Test output attributes and its types for table");
+
+        SiddhiManager siddhiManager = new SiddhiManager();
+
+        String streams = "" +
+                "define stream StockStream (symbol string, price float, volume long);" +
+                "@PrimaryKey('symbol') " +
+                "define table StockTable (symbol string, price float, volume long); ";
+        String storeQuery = "" +
+                "from StockTable " +
+                "select * ;";
+
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(streams);
+
+        siddhiAppRuntime.start();
```

Comment:
You have to stop the siddhiApp started.

Reply:
@NisalaNiroshana  ping! This PR is staling